### PR TITLE
tar: remove last example to reduce count to 8

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page has 8 or fewer examples.

For #5046

### CODE REVIEW

**Comment by bl-ue** [comment]
I picked the last example because it looked like a useful one, but the user has to specify the `--strip-components` option and it may not be obvious what to pass it.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Comment by bl-ue** [comment]
@owenvoke why didn't you merge this?  Can I?

**Comment by owenvoke** [comment]
Apologies, I forgot to merge.

**Comment by bl-ue** [comment]
Don't apologize!  I was just wondering if we have some special procedure I was missing.

**Comment by waldyrious** [comment]
For future reference, this example was initially introduced in #3859.

@bl-ue, good question about the self-merge. Our [maintenance guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md#ii-handling-prs) does not explicitly address this case, but technically the first rule of handling PRs would allow for a self-merge after two approvals by other maintainers. That said, IMO asking first (as you did) is the right call, since there may be indeed a reason why the merge wasn't performed.

**Comment by bl-ue** [comment]
Okay, thank you @waldyrious!  I should have probably checked when that example was added before removing it :)

That's good info to have.

**Comment by waldyrious** [comment]
> I should have probably checked when that example was added before removing it :)

Just to be clear, I didn't mean to criticize you for not doing that :) I just thought it would be useful to add that info, for historical reference.

**Comment by bl-ue** [comment]
No worries @waldyrious! I'm not offended :)

**Pull request merged**
